scvelo/tools/_em_model.py

```diff
@@ -126,6 +119,23 @@ def __init__(
         self._n_jobs = get_n_jobs(n_jobs=n_jobs)
         self._backend = backend
 
+    def _initialize_state_dict(
```

Comment:
Why do we need the argument `parameters` and `key` if we never pass them to the function?